package.json

```diff
@@ -28,7 +28,6 @@
     "lint:js": "eslint src/ --fix",
     "lint:scss": "stylelint 'src/**/*.scss' --fix",
     "prepublishOnly": "npm-run-all -s test:ci lint build lib",
-    "prepare": "npm run snyk-protect",
```

Comment:
minor: let's add a comment here about the temporary deletion of snyk-protect so we won't forget to put this back once the snyk issues are resolved on their end :D

Reply:
I'm logging a Tech Improvement ticket for that.